app/src/main/java/org/oppia/app/testing/NavigationDrawerTestActivity.kt

```diff
@@ -12,6 +15,13 @@ class NavigationDrawerTestActivity : InjectableAppCompatActivity(), RouteToTopic
   @Inject
   lateinit var homeActivityPresenter: HomeActivityPresenter
```

Comment:
Convert this to single line:
`@Inject lateinit var homeActivityPresenter: HomeActivityPresenter`

Reply:
Done.